mapswipe_workers/mapswipe_workers/generate_stats/project_stats.py

```diff
@@ -118,11 +118,19 @@ def get_tasks(filename: str, project_id: str) -> pd.DataFrame:
 
         sql_query = sql.SQL(
             """
-            COPY (
-                SELECT project_id, group_id, task_id, ST_AsText(geom) as geom
-                FROM tasks
-                WHERE project_id = {}
-            ) TO STDOUT WITH CSV HEADER
+COPY (
+    SELECT project_id, group_id, task_id, ST_AsText(geom) as geom,
+     (project_type_specifics->'properties'->'osmId')::text as osmId,
```

Comment:
interesting, this seems to work also for normal project types and the value for osmId will just be `NULL` in such situations. However, this will look a bit strange in the csv file.

Ideally we can check the project type here and then either use the old query or in case it is a "building footprint" project then we use your query.

Reply:
or maybe you can check in the write_sql_to_gzipped_csv function if you can simply drop the empty columns, so that they don't show up in the csv file as well.